include/modem/location.h

```diff
@@ -494,6 +521,28 @@ void location_cellular_ext_result_set(
 	enum location_cellular_ext_result result,
 	struct location_data *location);
 
+/**
+ * @brief Pass Wi-Fi location result to the library.
+ *
+ * @details If the Location library is not receiving Wi-Fi position directly from services,
+ * it triggers the @ref LOCATION_EVT_WIFI_EXT_REQUEST event when access point information
+ * should be sent to cloud services for location resolution. Then, the application responds
```

Comment:
Then, the application responds with the result of the location result. <- sentence does not make sense